app/api/discount_codes.py

```diff
@@ -70,6 +73,63 @@ def validate_quantity(self, data, original_data):
                          type_='event')
 
 
+class DiscountCodeSchemaEvent(Schema):
```

Comment:
Oh okay, That could be an issue. I also checked docs to see if any option to drop attributes or redefine them. There is an option to load only specific attributes but I guess it is from SqlAlchemy Model.